web/spacewalk-web.spec

```diff
@@ -40,18 +40,20 @@ Group:          Applications/Internet
 Version:        4.2.4
 Release:        1%{?dist}
 Url:            https://github.com/uyuni-project/uyuni
-Source0:        https://github.com/spacewalkproject/spacewalk/archive/%{name}-%{version}.tar.gz
+Source0:        https://github.com/uyuni-project/uyuni/archive/%{name}-%{version}.tar.gz
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 BuildArch:      noarch
 Requires(pre):  uyuni-base-common
 BuildRequires:  uyuni-base-common
 BuildRequires:  perl(ExtUtils::MakeMaker)
 BuildRequires:  susemanager-nodejs-sdk-devel
+BuildRequires:  nodejs-packaging
 
 %if 0%{?suse_version}
 BuildRequires:  apache2
-BuildRequires:  nodejs-packaging
-
+BuildRequires:  nodejs-common
```

Comment:
For TW, the current version of node is `nodejs-default` provided by nodejs-common sources. I will submit this update to SLE so this can be used in the future. For now, the only sure way is to BR: on nodejs10 (if you need 32bit) or nodejs14 (it's available in all supported non-LTSS codestreams)

Reply:
@sbluhm I think this looks good with the above suggestion applied. I think that Node 10 is a good target for now since that's what our pipeline tests against as well, I'll do the update everywhere at once when we're ready to move forward. Could you please also fix the typo of `nose` → `node` in the changefile?

Reply:
@Etheryte and @AdamMajer, I am unsure what to put. As far as I understand we should keep nodejs-common as it includes nose (purpose of this PR). It also can provide nodejs-default for TW or SLE in future. So I don't see a code change requirement. Or do you want me to put nodejs10/nodejs14 instead of nodejs-common?

I intentionally put "nose" in the change log as this is the command where CenOS is failing/is missing.

Reply:
It would be better to list specific nodejs version, like nodejs14 or nodejs10 or another. This can be changed later.